src/pages/index.tsx

```diff
@@ -4,29 +4,154 @@ import Layout from '@theme/Layout';
+              Get Started!
+            </Link>
+            <Link className="button button--secondary" to="/docs/introduction">
+              Explore the docs
+            </Link>
+          </div>
+        </div>
+      </div>
+    </section>
+  );
+}
+
+function OpenSource() {
+  return (
+    <section className={clsx(styles.explainerSection, styles.tint)}>
+      <div className={styles.twoColumns}>
+        <div className={styles.twoColumns__column}>
+          <h2>Always Open</h2>
+          <p>At blindnet, we strongly think Openness and Privacy are two sides of the same coin.</p>
+          <p>
+            We meticulously follow the{' '}
+            <a href="https://github.com/blindnet-io/openness-framework/blob/main/principles.md" target="_blank" rel="noopener noreferrer">
+              Openness Principles
+            </a>{' '}
+            in all matters, guided by the{' '}
+            <a href="https://github.com/blindnet-io/openness-framework" target="_blank" rel="noopener noreferrer">
+              Openness Framework
+            </a>{' '}
+            we defined and constantly update.
+          </p>
+          <p>
+            Obviously, this means all the core components of blindnet devkit and privacy stack are Open Source, following an Open Core model. But we go further than that: except when absolutely needed, all the contents we build and work with are and will always stay available to anyone and open to contribution. This includes every important <a href="https://github.com/blindnet-io/openness-framework/tree/main/DecisionFramework" target="_blank" rel="noopener noreferrer">decision</a> we take and every <a href="https://github.com/blindnet-io/product-management">specification and guiding document</a> we work with.
+          </p>
+          <p>
+            Making all matters easily accessible and intelligible to everyone is a top priority at blindnet. If you feel any information is missing or is incomplete, please <a href="https://github.com/blindnet-io/devrel-management/issues/new?assignees=noelmace&labels=request%2Ctriage&template=request.yml&title=%5BRequest%5D%3A+" target="_blank" rel="noopener noreferrer">let us know</a>!
+          </p>
+        </div>
+        <div className={styles.twoColumns__column}>
+          <OpenSourceSVG className={styles.twoColumns__illustration} />
+        </div>
+      </div>
+    </section>
+  );
+}
+
+function Construction() {
+  return (
+    <section className={styles.explainerSection}>
+      <div className={styles.twoColumns}>
+        <div className={styles.twoColumns__column}>
+          <ScienceSVG className={styles.twoColumns__illustration} />
+        </div>
+        <div className={styles.twoColumns__column}>
+          <h2>Constantly Innovating</h2>
+          <p>
+            Our team of PhDs, engineers, DevX experts and developers is constantly working on new ways to improve and extend the blindnet devkit.
```

Comment:
Software **engineer** and **developer** are often interchangeable terms. I suggest to drop one or add e.g. **security engineers**, **software developers**.